examples/basic/stream_function_call_args.py

```diff
@@ -0,0 +1,222 @@
+import asyncio
```

Comment:
If we have examples like this, we prefer having responses api model. Also, this example is so good! but it's a bit lengthy. So, more simplified version would be better to have.